### ISSUE TITLE
Scrollable regions are not accessible by keyboard  /exercises-automation-npm.html

### ISSUE THREAD

**Issue by Moro-Code** [sara-sabr/ITStrategy#727]
**Description**
relates to #715 

**Device Information**  
*Device*:  
*Operating System (incl. version)*:  
*Browser (incl. version)*:  
*Additional Details*:

**Element(s) Information**  
*Page*:  https://sara-sabr.github.io/ITStrategy/exercises-automation-npm.html
*CSS Selector(s) (if possible and applicable)*:  
*Additional Identifier(s) (if possible and applicable)*: All code sections   
*Additional Details*:  

**Screenshot(s) (if possible)**

**Additional information or thoughts**

https://dequeuniversity.com/rules/axe/3.4/scrollable-region-focusable?application=AxeChrome
